GrandCentralBoard/Widgets/Harvest/View/AreaBarChartComponentViewModel+FromDailyBillingStats.swift

```diff
@@ -137,3 +128,18 @@ extension NSDate {
         return formatter.stringFromDate(self)
     }
 }
+
+
+extension Double {
+    var formattedString: String {
+        let intValue = Int(self)
+        let isInteger = Double(intValue) == self
+
+        if isInteger {
+            return String(intValue)
+
+        } else {
```

Comment:
Then you should also have opening and closing bracket on the same indentation level and so on...